File: matiasmasso/Mat.NET/DT2/DTORepeticio.vb
Public Class DTORepeticio
    Inherits DTOContact

    Property Qty As Integer
    Property Orders As Integer
    Property Eur As Decimal

    Public Sub New()
        MyBase.New()
    End Sub

    Public Sub New(oGuid As Guid)
        MyBase.New(oGuid)
    End Sub

    Shared Function All(items As List(Of DTORepeticio), iOrderCount As Integer) As List(Of DTORepeticio)
        Dim retval As List(Of DTORepeticio) = items.Where(Function(x) x.Orders = iOrderCount)
        Return retval
    End Function

End Class

File: matiasmasso/Mat.NET/DTO/DTORepeticio.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace DTO
{
    public class DTORepeticio : DTOContact
    {
        public int Qty { get; set; }
        public int Orders { get; set; }
        public decimal Eur { get; set; }

        public DTORepeticio() : base()
        {
        }

        public DTORepeticio(Guid oGuid) : base(oGuid)
        {
        }

        public static List<DTORepeticio> All(List<DTORepeticio> items, int iOrderCount)
        {
            List<DTORepeticio> retval = (List<DTORepeticio>)items.Where(x => x.Orders == iOrderCount);
            return retval;
        }
    }
}
